src/pytypes/onefuzztypes/models.py

```diff
@@ -399,6 +399,8 @@ class Node(BaseModel):
     scaleset_id: Optional[UUID] = None
     tasks: Optional[List[Tuple[UUID, NodeTaskState]]] = None
     version: str = Field(default="1.0.0")
+    reimage_requested: bool = Field(default=False)
+    delete_requested: bool = Field(default=False)
```

Comment:
could both of these have been part of the node states

Reply:
We did that previously, but then got into issues where we were out of sync with the supervisor.  This treats the supervisor as having ground truth knowledge of node.state, but still lets us remember when a node should be reimaged/deleted.